scripts/src/actions/fix-yaml-config.ts

```diff
@@ -1,4 +1,6 @@
 import 'reflect-metadata'
+import {addFileToAllRepos} from './shared/add-file-to-all-repos'
 import {format} from './shared/format'
 
+addFileToAllRepos('.github/workflows/stale.yml', '.github/workflows/stale.yml')
```

Comment:
should `addFileToAllRepos` default to/from to the same path so we don't have this clunky interface of having to pass the same path twice?

Reply:
Good idea, let me add this.

Reply:
Done in https://github.com/protocol/github-mgmt-template/commit/47fefec4f514a564c9d8002b41bb7d6b6df51f7e

We'll be able to simplify this file after we upgrade GitHub Management here.